docs/040_pepr-tutorials/020_create-pepr-dashboard.md

```diff
@@ -0,0 +1,154 @@
+# Tutorial - Create a Pepr Dashboard
+
+## Introduction
+
+This tutorial will walk you through the process of creating a dashboard to display your Pepr metrics. This dashboard will present data such as the number of validation requests processed, the number of mutation requests that were allowed, the number of errors that were processed, the number of alerts that were processed, the status of the Pepr pods, and the scrape duration of the Pepr pods. This dashboard will be created using [Grafana](https://grafana.com/). The dashboard will display data from [Prometheus](https://prometheus.io/), which is a monitoring system that Pepr uses to collect metrics.
+
+This tutorial is not intended for production, but instead is intended to show how to quickly scrape Pepr metrics. The [kube prometheus stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) provides a more production suitable way of deploying Prometheus in prod.
+
+An example of what the dashboard will look like is shown below:
+
+![Pepr Dashboard](../../dashboards/pepr-dashboard-screenshot.png)
+
+***Note:*** *The dashboard shown above is an example of what the dashboard will look like. The dashboard will be populated with data from your Pepr instance.*
+
+## Prerequisites
+
+1. Have a Pepr module created and deployed. If you do not have a Pepr module created and deployed, follow the [Create a Pepr Module](010_create-pepr-module.md) tutorial.
+2. Have a local Kubernetes cluster running. If you do not have a local Kubernetes cluster running, follow the [Create a Pepr Module](010_create-pepr-module.md) tutorial.
+
+## Steps
+
+### **Create a Prometheus deployment**
+
+#### Apply the CRDs to your cluster
+
+    ```bash
+    kubectl create -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/main/bundle.yaml
+    ```
+
+    This command creates in the `default` namespace the necessary Custom Resource Definitions that the Prometheus Operator will use.
+
+#### Apply the [Prometheus Operator deployment](../../dashboards/prometheus-operator.yaml) to your cluster
+
+    ```bash
+    kubectl apply -f prometheus-operator.yaml
+    ```
+
+    This deploys the Prometheus Operator in your Kubernetes cluster.
```

Comment:
```suggestion
This deploys the Prometheus Operator in your Kubernetes cluster.
```

Reply:
The formatting is off on this markdown. The user should be able to quickly copy and paste the commands but the text is getting in the way. There are more instances of this in the doc. Other than this and  applying by file locations i think it is very good 

![image](https://github.com/defenseunicorns/pepr/assets/1096507/286240d5-1865-4bf0-97cc-9e0b1c3fc42a)